src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java

```diff
@@ -128,34 +128,45 @@ public void test() {
         Message<byte[]> messagePrIGM = output.receive(1000);
         assertEquals("TSO_IGM", messagePrIGM.getHeaders().get("type"));
         Message<byte[]> messageMergeStarted = output.receive(1000);
-        assertEquals("MERGE_STARTED", messageMergeStarted.getHeaders().get("type"));
-        Message<byte[]> messageMergeFinished = output.receive(1000);
-        assertEquals("MERGE_FINISHED", messageMergeFinished.getHeaders().get("type"));
+        assertEquals("MERGE_PROCESS_STARTED", messageMergeStarted.getHeaders().get("type"));
+
+        Message<byte[]> readNetworksStarted = output.receive(1000);
+        assertEquals("READ_NETWORKS_STARTED", readNetworksStarted.getHeaders().get("type"));
+        Message<byte[]> readNetworksFinished = output.receive(1000);
+        assertEquals("READ_NETWORKS_FINISHED", readNetworksFinished.getHeaders().get("type"));
+
+        Message<byte[]> mergeNetworksStarted = output.receive(1000);
+        assertEquals("MERGE_NETWORKS_STARTED", mergeNetworksStarted.getHeaders().get("type"));
+        Message<byte[]> mergeNetworksFinished = output.receive(1000);
+        assertEquals("MERGE_NETWORKS_FINISHED", mergeNetworksFinished.getHeaders().get("type"));
+        Message<byte[]> mergedNetworksStored = output.receive(1000);
+        assertEquals("MERGED_NETWORK_STORED", mergedNetworksStored.getHeaders().get("type"));
+
```

Comment:
Maybe, you can go further and test the **reception of messages** "LOAD_FLOW_STARTED", "LOAD_FLOW_FINISHED" and "MERGE_PROCESS_FINISHED" ?

Reply:
Done